packages/core/src/browser/core-preferences.ts

```diff
@@ -72,6 +72,18 @@ export const corePreferenceSchema: PreferenceSchema = {
             A setting of 'compact' will move the menu into the sidebar.`,
             included: !isOSX
         },
+        'window.titleBarStyle': {
+            type: 'string',
+            enum: ['native', 'custom'],
+            markdownEnumDescriptions: [
+                'Native title bar is displayed.',
+                'Custom title bar is displayed.'
+            ],
+            default: isWindows ? 'custom' : 'native',
+            scope: 'application',
+            markdownDescription: 'Adjust the appearance of the window title bar. Changes require a full restart to apply',
+            included: !isOSX
+        },
```

Comment:
@msujew we probably only want to support this preference in the desktop (electron) use-case? Should it be moved to an electron-only preference-schema?

Reply:
Yes, that makes sense 👍